strands_executive_msgs missing in in `build_depend`

see https://lcas.lincoln.ac.uk/jenkins/job/ros-indigo-wait-action_binarydeb_trusty_amd64/1/console

```
CMake Error at /opt/ros/indigo/share/catkin/cmake/catkinConfig.cmake:75 (find_package):
  Could not find a package configuration file provided by
  "strands_executive_msgs" with any of the following names:

    strands_executive_msgsConfig.cmake
    strands_executive_msgs-config.cmake

```

--- a/wait_action/package.xml
+++ b/wait_action/package.xml
@@ -35,6 +35,7 @@
   <build_depend>actionlib_msgs</build_depend>
   <build_depend>rospy</build_depend>
   <build_depend>std_srvs</build_depend>
+  <build_depend>strands_executive_msgs</build_depend>
   <run_depend>actionlib</run_depend>
   <run_depend>actionlib_msgs</run_depend>
   <run_depend>rospy</run_depend>

